components/Header.js

```diff
@@ -12,13 +12,19 @@ class Header extends Component {
     this.state = { menuHidden: true };
   }
 
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (this.state.menuHidden !== prevState.menuHidden) {
+      this.props.onMenuToggle(this.state.menuHidden);
+    }
+  }
+
   handleMenuToggle = e => {
     // console.log(e.target);
-    this.setState(prevState => ({
-      menuHidden: !prevState.menuHidden,
-    }));
-
-    this.props.onMenuToggle();
+    if (window.innerWidth <= parseInt(this.props.theme.medium)) {
```

Comment:
I noticed that if I open the mobile menu and then make my browser window wider, I'm not able to scroll because of the `overflow: hidden;`. That makes me think that we should use a CSS media query to make sure that [`.header-menu-open`](https://github.com/texas-justice-initiative/website-nextjs/blob/2332a14bc2c7e2fd059807a86b6090787b817f7d/components/Page.js#L47-L50) has no impact when we're looking at the non-mobile version of the site.

If we did that, would we need this check?

Reply:
I tested this out and I think turning this into a media query is a nice solution to this issue as well.

Reply:
Whoops, sorry I merged before I saw all the comments... I'm adding the media query too, but I think it's also bad for header-menu-open to be applied when the header menu is not open. If we want to associate anything else with that class later then we might run into as similar problem to this one.